datumaro/plugins/splitter.py

```diff
@@ -150,16 +166,31 @@ def __iter__(self):
             yield self.wrap_item(item, subset=self._find_split(i))
 
 
-class ClassificationSplit(_TaskSpecificSplit):
+class ClassificationSplit(_TaskSpecificSplit, CliPlugin):
     """
     Splits dataset into train/val/test set in class-wise manner. |n
+    In other words, the split ratio of each class is preserved
+    as much as possible.|n
     |n
     Notes:|n
     - Single label is expected for each DatasetItem.|n
     - If there are not enough images in some class or attributes group,
       the split ratio can't be guaranteed.|n
+    |n
+    Example:|n
+    |s|s%(prog)s --subset train:.5 --subset val:.2 --subset test:.3
     """
 
+    @classmethod
+    def build_cmdline_parser(cls, **kwargs):
```

Comment:
You can pull this function to the base class.

Reply:
Oh, I see.